environments/centos7/Vagrantfile

```diff
@@ -1,14 +1,19 @@
 Vagrant.configure("2") do |config|
   config.vm.box = "generic/centos7"
-  config.vm.provision "file", source: "../../../environments/etc", destination: "$HOME/puppet"
+  config.vm.provision "file", source: "../../environments/etc", destination: "$HOME/puppet"
+  config.vm.synced_folder "../../", "/puppet-datadog-agent"
   config.vm.provision "shell", inline: <<-SHELL
-    rpm -ivh http://yum.puppetlabs.com/puppetlabs-release-el-7.noarch.rpm
+    #install puppet 5
+    rpm -ivh http://yum.puppetlabs.com/puppet5-release-el-7.noarch.rpm
     yum install -y puppet
-    systemctl start puppet
-    systemctl enable puppet
+    ln -s /opt/puppetlabs/puppet/bin/puppet /usr/bin/puppet
 
-    gem install r10k -v 2.6.7
+    # install modules
     cd /home/vagrant/puppet
-    sudo su vagrant -c "/usr/local/bin/r10k puppetfile install --moduledir=/tmp/modules"
```

Comment:
it's not that we're calling vagrant, it's that we do stuff as the vagrant user, to create files with the correct permissions

Reply:
True. I didn't read it properly.

I use root once inside the VM so that's why I didn't care about permissions. I guess it's fine since it's a test env, but maybe we should say it in the README. Or even better: tell vagrant to `ssh` as root by default.